### PR TITLE
[FIX] project: add missing stage when import tasks

### PR DESCRIPTION
**Pull request by RaoufGhrissi**
Before this commit:
Let's consider project P1 having no stages.
Let's consider project P2 having Stage 1, Stage 2
When importing tasks via csv file:
- Task 1 for project P1 and Stage 1
- Task 2 for project P1 and Stage 2

Stage 1 and Stage 2 are still assigned only to project P1.

After this commit:
Stage 1 and Stage 2 are now assigned to P1 and P2.

Task-2996393

### CODE REVIEW

File: addons/project/models/project.py [hunk 1]
```diff
@@ -1096,6 +1096,18 @@ def create(self, vals_list):
                 task._portal_ensure_token()
         return tasks
 
+    def _load_records_create(self, values):
+        tasks = self.create(values)
```
Comment (auon-odoo) [inline addons/project/models/project.py:1100]: You should call super

File: addons/project/tests/test_project_base.py [hunk 2]
```diff
@@ -104,3 +104,30 @@ def test_delete_project_with_tasks(self):
...
+        self.assertEqual(len(self.project_goats.type_ids), 2)
+        first_stage, second_stage = self.project_goats.type_ids
+        self.env['project.task'].create({
+            'name': 'First Task',
+            'user_id': self.user_projectuser.id,
+            'project_id': self.project_pigs.id,
+            'stage_id': first_stage.id
+        })
+        self.assertEqual(self.project_pigs.type_ids, first_stage)
+        self.env['project.task'].create([
```
Comment (auon-odoo) [inline addons/project/tests/test_project_base.py:119]: You should test _load_records_create

File: addons/project/tests/test_project_base.py [hunk 3]
```diff
@@ -104,3 +104,30 @@ def test_delete_project_with_tasks(self):
...
+                'stage_id': second_stage.id
+            },
+            {
+                'name': 'Third Task',
+                'user_id': self.user_projectuser.id,
+                'project_id': self.project_pigs.id,
+                'stage_id': first_stage.id
+            },
+        ])
+        self.assertEqual(self.project_pigs.type_ids, self.project_goats.type_ids)
```
Comment (auon-odoo) [inline addons/project/tests/test_project_base.py:133]:
```suggestion
        self.env['project.task']._load_records_create([
            {
                'user_id': self.user_projectuser.id,
                'project_id': self.project_pigs.id,
                'stage_id': first_stage.id
            } for stage in self.project_goats.type_ids
        ])
        self.assertEqual(self.project_pigs.type_ids, self.project_goats.type_ids)
```
Isn't this enough?
Btw, I don't remember how data is supposed to be contained in the `values` param of _load_records_create. So it would have to be checked.

File: addons/project/tests/test_project_base.py (hunk 2 above)
Comment (auon-odoo) [reply addons/project/tests/test_project_base.py:119]: When you write a test, you should check that it fails without your fix. Iinm, it is not the case here.

**Review by auon-odoo — CHANGES_REQUESTED** [review]
Left some comments again. After that, it should be good.

File: addons/project/tests/test_project_base.py (hunk 2 above)
Comment (RaoufGhrissi) [reply addons/project/tests/test_project_base.py:119]: it's already the case, if i comment my fix, the test fail 
image

File: addons/project/tests/test_project_base.py (hunk 3 above)
Comment (RaoufGhrissi) [reply addons/project/tests/test_project_base.py:133]: name is mandatory but okay i will give them the same one

Comment (RaoufGhrissi) [reply addons/project/tests/test_project_base.py:133]: checked, it the same as create

**Review by RaoufGhrissi — COMMENTED** [review]
Thank you for your review, @auon-odoo fixed

**Review by auon-odoo — APPROVED** [review]
lgtm :)

File: addons/project/models/project.py [hunk 4]
```diff
@@ -1175,7 +1187,7 @@ def _compute_partner_id(self):
                 if task.project_id.partner_id:
                     task.partner_id = task.project_id.partner_id
             else:
-                task.partner_id = task.project_id.partner_id or task.parent_id.partner_id 
+                task.partner_id = task.project_id.partner_id or task.parent_id.partner_id
```
Comment (xavierbol) [inline addons/project/models/project.py:1190]: unnecessary diff 🙂

**Review by xavierbol — COMMENTED** [review]
LGTM, thanks for the fix.

However, for the commit message is more a bug instead of improvement.
Can you reword the commit title accordingly? 🙂 

Something like:
`[FIX] project: add missing stage when import tasks`

**Review by xavierbol — APPROVED** [review]
LGTM, thanks for the fix. 🙂 

robodoo r+
